test/validate/eia_test.py

```diff
@@ -10,16 +10,17 @@
 
 @pytest.mark.parametrize(
     "df_name,cols", [
-        ("plants_eia860", "all"),
-        ("utils_eia860", "all"),
-        ("pu_eia860", "all"),
+        ("bf_eia923", "all"),
```

Comment:
are these just reorganized to get whatever failure you were getting to show up early?? or are they all now just alphabetized?

Reply:
I alphabetized these lists, and also added new output tables to them, like `gfn_eia923` and `all_plants_ferc1`